CHANGELOG.md

```diff
@@ -15,7 +15,8 @@ This changelog keeps track of work items that have been completed and are ready
 
 ### Breaking Changes
 
-- **General**: TODO ([#TODO](https://github.com/kedacore/http-add-on/issues/TODO))
+- **General**: Add multi-host support to `HTTPScaledObject` ([#552](https://github.com/kedacore/http-add-on/issues/552))
```

Comment:
Let's move this to Deprecations as it isn't a breaking change now

Reply:
Done :)